WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -534,6 +536,9 @@ extension ProductFormViewController: UITableViewDelegate {
             case .categories:
                 // TODO-2000 Edit Product M3 analytics
                 editCategories()
+            case .tags:
+                // TODO-2081 Edit Product M3 analytics
```

Comment:
nit: I just created an issue for M3 analytics if you'd like to link it here (TODO-2509) since we usually work on them toward the end :) https://github.com/woocommerce/woocommerce-ios/issues/2509

Reply:
Updated here ;) f635e339d95650664bbeda87a2d1704f551920d0